microprofile/microprofile-config-1.3/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -1,5 +1,2 @@
-app.timeout=314159
-fileProperty=This will be overriden by a file
-yamlOrdereredProperty=Property from default config source
```

Comment:
Why is this deleted?

Reply:
3 unused properties removed, 2 remaining are used